systemtests/tests/checkpoints/testrunner-checkpoints-on-kill

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+#
+# Kill the daemons while running a backup.
+# Check that metadata is still saved with checkpoints.
+#
+TestName="$(basename "$(pwd)")"
+export TestName
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+
+start_test
+
+slowjob="slow-backup-bareos-fd"
+
+for daemon in "dir" "sd" "fd"; do
+
+    backup_log=$tmp/kill-$daemon-backup-checkpoints.out
+    restore_log=$tmp/kill-$daemon-restore-checkpoints.out
+    restore_directory=$tmp/kill-$daemon-checkpoint-restores
+
+    rm -f $backup_log
+    rm -f $restore_log
+
+cat <<END_OF_DATA >"$tmp/bconcmds"
+@$out /dev/null
+messages
+@$out $backup_log
+run job=$slowjob level=Full yes
+quit
+END_OF_DATA
+
+    run_bconsole
+
+    sleep 17
```

Comment:
Same here, fixed wait time is bad, as it can fail on very busy systems or waste time on quick systems